src/Razor/tools/RazorExtensionsTransport/RazorExtensionsTransport.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename this project to create a more distinctive package e.g. Microsoft.AspNetCore.Razor.Internal.csproj